src/components/card-with-image/card-with-image.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CardWithImage extends React.Component {
```

Comment:
I agree with the idea in your comment that these two components could be effectively merged into one.